modules/src/what_cocktail.py

```diff
@@ -0,0 +1,351 @@
+#rewrite the program (which a burglar stole from me!) and then release it onto telegram and messenger!
+''' argh what cocktail can i make with these ingredients? search and return a cocktail which has
+the most matching ingredients'''
+from enchant.checker import SpellChecker
+import re
+
+print("What? Cocktail")
+print()
+print()
+
+def search(drinks, dictionary):
+    filter_dict = {}
+    for key, value in dictionary.items():
+        for v in value:
+            if v == drinks:
+                filter_dict.update({key: value})
+    drinks = filter_dict
```

Comment:
You could directly return `filter_dict` instead of assigning it to drinks. It has no impact on the code.

Reply:
done

Reply:
👌